lib/stack_master/parameter_resolvers/parameter_store.rb

```diff
@@ -0,0 +1,24 @@
+module StackMaster
+  module ParameterResolvers
+    class ParameterStore < Resolver
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def resolve(value)
+        resp = ssm.get_parameter(
+          name: value,
+          with_decryption: true
+        )
+        resp.parameter.value
```

Comment:
Do you know how array parameter values are handled by StackMaster? Does a `StringList` parameter type need to be transformed into an `Array` or will it be handled just fine by StackMaster?

Reply:
No idea,  I'll add that to the todo list to figure out.

Reply:
I don't actually thing it's an issue here,  there is only ever one parameter with the same name, so you can only get one value, so iterating and fetching isn't sensible.  (which is how the other array_parameters work) 
That said if you need to you could store "1,2,3,4,5" as the value for the parameter and it will just work as all stackmaster does is concatenate with comma separators (at least according to my reading of https://github.com/envato/stack_master/blob/master/lib/stack_master/resolver_array.rb#L9-L13)

Reply:
Yeah, that's my understanding also.